src/autotune/files/autotune.py

```diff
@@ -15,34 +15,21 @@
 import atexit
 import os
 import platform
-import re
 import shlex
 import subprocess
 import sys
 
-sys.path.append('/usr/local/www')
-sys.path.append('/usr/local/www/freenasUI')
+from middlewared.client import Client
 
-os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'freenasUI.settings')
-
-import django
-django.setup()
-
-from freenasUI.middleware.notifier import notifier
-n = notifier()
-if n.is_freenas():
+c = Client()
```

Comment:
We should close it ?

Reply:
i thought about it, but its not a long running script, so it should be fine and be taken care of during exit.